File: PRÁCTICA SEGUNDO PARCIAL/Ejercicio15/Java/VectorDigitos.java
import java.util.ArrayList;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Ingrese un número: ");
        int numero = scanner.nextInt();

        ArrayList<Integer> digitosImpares = new ArrayList<>();
        ArrayList<Integer> digitosPares = new ArrayList<>();

        String numeroStr = Integer.toString(numero);
        for (int i = 0; i < numeroStr.length(); i++) {
            int digito = Character.getNumericValue(numeroStr.charAt(i));
            if (digito % 2 == 0) {
                digitosPares.add(digito);
            } else {
                digitosImpares.add(digito);
            }
        }

        System.out.println("Dígitos pares: " + digitosPares);
        System.out.println("Dígitos impares: " + digitosImpares);
    }
}

File: PRÁCTICA SEGUNDO PARCIAL/Ejercicio15/Python/VectorDigitos.py
# Con input
numero = input("Ingrese un número: ")

digitos_impares = []
digitos_pares = []

for digito in numero:
    if int(digito) % 2 == 0:
        digitos_pares.append(int(digito))
    else:
        digitos_impares.append(int(digito))

print("Dígitos pares:", digitos_pares)
print("Dígitos impares:", digitos_impares)
